k8s/squest_k8s/tasks/02-db.yml

```diff
@@ -191,8 +192,8 @@
             selector:
               app.kubernetes.io/name: squest-phpmyadmin
 
-    - when: squest_phpmyadmin is defined and squest_phpmyadmin.ingress is defined and squest_phpmyadmin.ingress.enabled
```

Comment:
we actually like to keep "when" in first lines to mock a standard language with a "if"